### ISSUE TITLE
Error when running write_nnet_test.jl

### ISSUE THREAD

**Issue by SebastianGuadalupe** [sisl/NeuralVerification.jl#124]
While running the file `runtests.jl`, it errors in the file `write_nnet_test.jl` with the following error:
```julia
Read nnet test: Error During Test at /home/sguadalupe/.julia/dev/NeuralVerification/test/write_nnet_test.jl:2
  Got exception outside of a @test
  UndefVarError: Layer not defined
  Stacktrace:
   [1] top-level scope at /home/sguadalupe/.julia/dev/NeuralVerification/test/write_nnet_test.jl:5
   [2] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113
   [3] top-level scope at /home/sguadalupe/.julia/dev/NeuralVerification/test/write_nnet_test.jl:5
   [4] include(::String) at ./client.jl:439
   [5] top-level scope at /home/sguadalupe/.julia/dev/NeuralVerification/test/runtests.jl:24
   [6] include(::Module, ::String) at ./Base.jl:377
   [7] exec_options(::Base.JLOptions) at ./client.jl:288
   [8] _start() at ./client.jl:484
```

**Comment by tomerarnon** [comment]
Strange, since the tests pass for me and with travis, including in #125 . Are you running the file with `include` or with `]test`?

**Comment by castrong** [comment]
If the issue comes when including the testing file (instead of with ]test) I just made PR #128 which should address that. It adds "NeuralVerification." in front of several objects so that it can be run with include as well.

**Comment by tomerarnon** [comment]
closed by #128
